.circleci/config.yml

```diff
@@ -118,21 +119,8 @@ jobs:
 
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - homebrew_cache_v14
-      # Bump cache version when changing this.
-      - run: echo 'export HOMEBREW_PACKAGES="go@1.17"' >> $BASH_ENV
-      # Only update when brew doesn't know about some of the packages because:
-      # 1) not worth the cost, and
-      # 2) hits github in a way that leads to flakyness
-      - run: brew info ${HOMEBREW_PACKAGES} || brew update || true
-      - run: HOMEBREW_NO_AUTO_UPDATE=true brew install ${HOMEBREW_PACKAGES}
-      - save_cache:
-          paths:
-            - /usr/local/Homebrew
-          key: homebrew_cache_v14
-      - run: echo 'export PATH="/usr/local/opt/go@1.17/bin:$PATH"' >> $BASH_ENV
+      - go/install:
+          version: "1.17.2"
```

Comment:
can you also add cache: true? https://circleci.com/developer/orbs/orb/circleci/go

Reply:
oh wait nm cache: true is the default